FirebaseFunctions/Sources/Functions.swift

```diff
@@ -139,11 +142,19 @@ internal enum FunctionsConstants {
    * Creates a reference to the Callable HTTPS trigger with the given name.
    * - Parameter name The name of the Callable HTTPS trigger.
    */
-  @objc(HTTPSCallableWithName:) open func httpsCallable(_ name: String) -> HTTPSCallable {
+  @objc(HTTPSCallableWithName:useLimitedUseAppCheckToken:) open func httpsCallable(_ name: String,
```

Comment:
This is an ObjC breaking change.  The old API must be kept for Objective C at least.

Reply:
Reverted this break and made the API change strictly additive.